.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
```

Comment:
Should we have a Pull request template?  

For issues i get why we should use templates to prevent lazy issues but for most people once they put work into a project they automatically also add a decent pull request information.

Reply:
Agreed, I've removed it.